orttraining/orttraining/core/graph/gradient_builder_base.h

```diff
@@ -295,6 +295,10 @@ class GradientBuilderBase {
     return ConstantScalarNode(1.0f, "OneConstant_Type" + std::to_string(elem_type), elem_type);
   }
 
+  static NodeDef MinusOneConstantNode(int elem_type) {
```

Comment:
We can do (0 - Sin(I(0)) * GO(0) so we don't need this MinusOneConstant (we already have ZeroConstant), and maybe from And/Sub is faster than Mul/Div in computing.

Reply:
Done!